addons/udes_stock_picking_batch/views/stock_picking_batch.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+
+        <!-- Stock picking batch form -->
+        <record id="stock_picking_batch_form" model="ir.ui.view">
+            <field name="name">udes_stock.stock.picking.batch.form</field>
+            <field name="model">stock.picking.batch</field>
+            <field name="inherit_id"
+                   ref="stock_picking_batch.stock_picking_batch_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='state']" position="replace">
+                    <field name="state" widget="statusbar" statusbar_visible="waiting,ready,in_progress,done"/>
+                </xpath>
+                <xpath expr="//button[@name='action_confirm']" position="after">
+                    <button name="mark_as_todo" string="Mark as Todo"
+                            confirm="Are you sure you want to mark this batch as todo?" type="object" states="draft"/>
+                </xpath>
+                <xpath expr="//button[@name='action_cancel']" position="replace">
+                    <button name="action_cancel" string="Cancel" type="object" states="ready,waiting,in_progress"/>
+                </xpath>
+            </field>
+        </record>
+
+        <!-- Stock picking batch filter -->
+        <record id="stock_picking_batch_filter" model="ir.ui.view">
+            <field name="name">udes_stock.stock.picking.batch.filter</field>
+            <field name="model">stock.picking.batch</field>
+            <field name="inherit_id"
+                   ref="stock_picking_batch.stock_picking_batch_filter"/>
+            <field name="arch" type="xml">
+                <xpath expr="//filter[@name='draft']" position="after">
+                    <filter name="ready" string="Ready"
+                            domain="[('state','=', 'ready')]"
+                            help="Batch Pickings Ready"/>
+                    <filter name="waiting" string="Waiting"
+                            domain="[('state','=', 'waiting')]"
+                            help="Batch Pickings Waiting"/>
+                </xpath>
+                <xpath expr="//filter[@name='in_progress']" position="replace">
```

Comment:
Can change what you need to change i think with position="attributes". I think replacing should be used as last resort.

example:

`<xpath expr="//filter[@name='in_progress']" position="attributes">`
    `<attribute name="help">Batch Transfers not finished</attribute>`
    `<attribute name="domain">[]</attribute>`
`</xpath>`

Reply:
What I wanted to change was the string, for some reason the code is still the same so I need to change the name. Happy to try your way anyway.

Reply:
Yes, better to change an individual attribute than replace the whole thing.

The original code has string as `In Progress`, but you've overridden it to still say `In Progress`. Am I missing something?

Reply:
@PeterC10 I wanted to change it to Running instead of "In Progress" since the display string for this state is "Running". Would that be a good idea?